CHANGELOG.md

```diff
@@ -196,5 +196,7 @@ can get data for both organization and organization brand pages.
   COMPANY_FOLLOWED_BY_MEMBER instead of CompanyFollowedByMember. 
   This changed in the LinkedIn API 202305 release.
 
-## 4.5.1 (Work in progress)
+## 4.5.1 (Feb 2, 2024)
```

Comment:
Remember to change this based on the day it actually gets merged/deployed?

Reply:
Actually I am going to remove the release date as I will merge the PR but not release